docs/change_log/2024-08-26-launching-activities-via-interactions.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Launching Activities in Response to Interactions"
```

Comment:
I am on the fence with having these as two separate changelog entries.  Really we're using this as an opportunity to introduce launching activities as interaction responses, and the entry point is required infra to make that work 🤔